frank/context.py

```diff
@@ -60,7 +60,8 @@ def inject_retrieval_context(alist: Alist, source) -> Alist:
         return alist
 
     context = alist.get(tt.CONTEXT)
-    context_store = {**context[0], **context[1], **context[2]}
```

Comment:
The context attribute is a list of dicts for:
- user context,
- environment context, and
- query context.
By default, the ctx attribute in an alist is `[{}.{},{}]`. If you are having issues with the context, then it could be that input queries do not have a properly defined context attribute value. Check to see if that's the source of the problem.

If that problem still persists, then you can replace the line you commented out with:
`context_store = {**context[0], **context[1], **context[2]} if context else {}`


If you are having issues using it, then, it could be that the